browsergym/experiments/src/browsergym/experiments/benchmark/base.py

```diff
@@ -55,6 +55,7 @@ class Benchmark(DataClassJsonMixin):
     high_level_action_set_args: HighLevelActionSetArgs
     is_multi_tab: bool
     env_args_list: list[EnvArgs]
+    full_reset_script: Optional[str]
```

Comment:
hmmm this is a symptom that we should have went with a class hieararchy instead of a single class for all benchmarks